lib/modules/onc_program/test/bulk_patient_export_sequence_test.rb

```diff
@@ -56,10 +56,10 @@
     before do
```

Comment:
As an aside, every time i see `before` I have to double check if it is before each example or just once for all examples, because it is important in this case because we are mutating `@conformance`.  I think it would be a good idea to use the explicit `before(:each)` form in general, but i know that's not what we do right now.